### PR TITLE
(lv2v) change control_url to not nullable

### PR DESCRIPTION
**Pull request by eliteprox**
This change modifies the `control_url` field to not-nullable. This change goes along with https://github.com/livepeer/go-livepeer/pull/3263

### CODE REVIEW

File: runner/app/routes/live_video_to_video.py [hunk 1]
```diff
@@ -118,6 +120,13 @@ async def live_video_to_video(
             ),
         )
 
+    if "trickle_port" in params.params:
+        trickle_port = params.params["trickle_port"]
```
Comment (leszko) [inline runner/app/routes/live_video_to_video.py:124]: Why do you need this? Shouldn't we pass correct port as part of `subscribe_url`/`publish_url`/`control_url`? I think that if we need to change it we should change it in O.

**Review by leszko — CHANGES_REQUESTED** [review]

File: runner/app/routes/live_video_to_video.py (hunk 1 above)
Comment (eliteprox) [reply runner/app/routes/live_video_to_video.py:124]: We can pass the port at the beginning of the URL instead and then append the host within the runner conditionally by checking for the colon and port number. I was trying to be more descriptive by explicitly passing the port with a named parameter, but I think either approach is good

Comment (leszko) [reply runner/app/routes/live_video_to_video.py:124]: To me it's super weird that you pass Url and then separately the port. So I'd prefer to not do that logic unless really needed.

Comment (leszko) [reply runner/app/routes/live_video_to_video.py:124]: Or maybe I'm missing something. But Afaiu all urls can already include the port so no need to pass it separately

Comment (eliteprox) [reply runner/app/routes/live_video_to_video.py:124]: The URL is passed without a host or port (example: `/ai/trickle/2d4a3F`). By using `request.client.host` we are able to reliably identify the host to use for following trickle updates, regardless of the serviceUri config of the orchestrator. This is why the block is added. However, I think we can use `:8936/ai/trickle/2d4a3F`, and append host when the value is passed starting with `:` if you'd like it a bit cleaner. This still allows for testing with a full URL.

Comment (eliteprox) [reply runner/app/routes/live_video_to_video.py]: Modified this to remove the `trickle_port` param entirely. It looks like we're going with ServiceURI. In the case a local development, an external host IP will need to be used.

**Pull request closed without merging**